plugin/keyring_vault/vault_keys_container.h

```diff
@@ -0,0 +1,29 @@
+#ifndef MYSQL_VAULT_KEYS_CONTAINER_H
+#define MYSQL_VAULT_KEYS_CONTAINER_H
+
+#include <my_global.h>
+#include <boost/core/noncopyable.hpp>
+#include "keys_container.h"
+#include "i_vault_io.h"
+
+namespace keyring
+{
+
+class Vault_keys_container : public Keys_container, private boost::noncopyable
+{
+public:
+  Vault_keys_container(ILogger* logger)
+    : Keys_container(logger)
+  {}
+
+  my_bool init(IKeyring_io* keyring_io, std::string keyring_storage_url);
```

Comment:
This function overloads keyring framework function - Keys_container::init function.